warrior/Tools/w_settings.xml

```diff
@@ -14,9 +14,9 @@
             3. every_failure - to send an email whenever a case fails during suite/project execution
          When 'mail_on attribute' is not given or left empty, default value will be used for sending email.
          It also accepts comma separated values, example mail_on="per_execution, first_failure" -->
-        <smtp_host></smtp_host> <!-- Optional, use FNC smtp host smtp.fnc.fujitsu.com to notify test case execution results -->
-        <sender></sender> <!-- Optional, use sender's email such as Warrior_PBO@fnc.fujitsu.com -->
-        <receiver></receiver> <!-- Optional, use receiver's email such as FNC.User@fnc.fujitsu.com -->
+        <smtp_host></smtp_host> <!-- Name of the host running your SMTP server(IP address or DNS name) -->
+        <sender></sender> <!-- Sender email id -->

```

Comment:
Do we need to give sender option even if it is optional? Again no impact on the overall review. 
Got this feedback while explaining the feature to a user. 

Good for removing Fujitsu specific info.